docs/ELF_PACKAGE_METADATA.md

```diff
@@ -14,7 +14,7 @@ or parse ELF core files.*
 
 ELF binaries get stamped with a unique, build-time generated hex string identifier called `build-id`,
 [which gets embedded as an ELF note called `.note.gnu.build-id`](https://fedoraproject.org/wiki/Releases/FeatureBuildId).
-In most cases, this allows to associate a stripped binary with its debugging information.
+In most cases, this allows one to associate a stripped binary with its debugging information.
```

Comment:
The changed version sounds overly formal. Please just rephrase this as
"this allows a stripped binary to be associated …"